src/views/IdentityOverview/IdentityOverview.tsx

```diff
@@ -126,6 +142,17 @@ export function IdentityOverview({ identity }: Props): JSX.Element | null {
         </Link>
       )}
 
+      {repairDid && (
+        <Link
+          to={generatePath(paths.identity.did.repair, {
+            address: identity.address,
+          })}
```

Comment:
We already have address in scope
```suggestion
          to={generatePath(paths.identity.did.repair, { address })}
```